.github/workflows/build-test.yaml

```diff
@@ -21,6 +21,19 @@ jobs:
 
       - run: go mod tidy -compat=1.17
 
+  sanity:
+    name: sanity
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.19
+        id: go
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: make test-sanity
```

Comment:
we should probably call this something more descriptive, like 'dont-change-manifest-file' or similar

Reply:
I thought on this one be like a sanity which can do more than on test check
In this case it is calling go vet (I notice that we are not checking this), we could for example ensure that all header has a license and etc. However, if we try to commit a change on the scripts/Manifest we will get the error message:

```
$ make test-sanity           
...
### WARNING: You cannot merge changes on the script/Manifest! ###
make: *** [test-sanity] Error 1
```

Reply:
But I will change that :-)